### DISCUSSION TITLE
Configure idle session timeout

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3979] [Help]
Hi budibase,

If I'm not mistaken current idle session is 24 hours in budibase, is there any way to configure the idle session so that user will redirect to login page after not active for more then 20 mins? 

Tq

**Comment by mjashanks** [comment]
This is not configurable at the moment.

**Reply by gpjorgenson** [reply]
This is something we need as well. Could it be put on a to do list?

**Comment by gvorster** [comment]
> Hi budibase,
> 
> If I'm not mistaken current idle session is 24 hours in budibase, is there any way to configure the idle session so that user will redirect to login page after not active for more then 20 mins?
> 
> Tq

Is it 24hrs? I have to login again after less than 12 hours.
